fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -40,9 +40,13 @@ statement
     | updateStatement                                                                       #update
     | deleteStatement                                                                       #delete
 
-    // Admin Set Statement
+    // Admin Statement
     | ADMIN SET FRONTEND CONFIG '(' property ')'                                            #adminSetConfig
     | ADMIN SET REPLICA STATUS properties                                                   #adminSetReplicaStatus
+    | ADMIN SHOW FRONTEND CONFIG (LIKE pattern=string)?                                     #adminShowConfig
+    | ADMIN SHOW REPLICA DISTRIBUTION FROM qualifiedName (PARTITION identifierList)?        #adminShowReplicaDistribution
+    | ADMIN SHOW REPLICA STATUS FROM qualifiedName (PARTITION identifierList)?
```

Comment:
Shall we add a show help command to display all these commands?

Reply:
Help statements can be provided, but that shouldn't be the job of this PR. And, currently our help command framework has been under maintenance